clusterprovider/kind/kind.go

```diff
@@ -85,6 +74,44 @@ func (k KindClusterProvider) List() ([]string, error) {
 	return logicalNameList, err
 }
 
+func (k KindClusterProvider) Get(ctx context.Context, lcName string) (clusterprovider.LogicalClusterInfo, error) {
+	logger := klog.FromContext(ctx)
+	cfg, err := k.kindProvider.KubeConfig(lcName, false)
+	if err != nil {
+		logger.Error(err, "couldn't fetch config for cluster")
```

Comment:
It's better to return the error. I don't think we can send an event if we don't have config. Return `nil` error in last `return`.